src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -96,9 +96,28 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
 
             String lowerFilePath = filePath.toLowerCase();
 
+
+
             if (!fileMap.containsKey(lowerFilePath)) {
                 LOG.warn("TsLint reported issues against a file that wasn't sent to it - will be ignored: " + lowerFilePath);
-                continue;
+
```

Comment:
So there's a bit of normalisation in there based on how tslint vs SonarQube report the path to be but I've not seen many issues so far. It does seem like an area where some work's needed though, to do proper normalisation between the SonarQube path and whatever the tooling is reporting and to handle the tooling reporting issues as relative paths (per @NikitaEgorov's fix above)